common-test/src/main/java/io/customer/commontest/BaseIntegrationTest.kt

```diff
@@ -1,7 +1,7 @@
 package io.customer.commontest
 
-import io.customer.sdk.CustomerIO
 import io.customer.sdk.CustomerIOConfig
+import io.customer.sdk.android.CustomerIO
```

Comment:
Because we haven't refactored common-test module yet and it uses older initialization.